src/assets/css/_fonts.scss

```diff
@@ -0,0 +1,16 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,400,400italic,500,500italic,700,700italic,900,900italic);
+@import url(https://fonts.googleapis.com/css?family=Roboto+Mono:100,100italic,300,300italic,400,400italic,500,500italic,700,700italic);
+
+// Add new font properties here.
+// The font property variable should be in the form $font-<type|size|...>-<domain>[-subdomain]
+
+$font-type-default: 'Roboto', 'Verdana', sans-serif;
+$font-type-code: 'Roboto Mono', 'Curier New', monospace;
+
+$font-size-default: 15pt;
+$font-size-code: 13pt;
+$font-size-header: 15pt;
+$font-size-footer: 15pt;
+$font-size-title: 15pt;
```

Comment:
Planen var å skille mellom elev/lærer-modus ved å bruke to forskjellige fargetemaer, grønn for elev og blå for lærere. Disse fargene skal brukes i mange forskjellige komponenter på alle sidene, f.eks navbar, knapper, oppgaver(oppgavetekst/oppgaveveiledning), filteret, footer... Her mener jeg det kunne være fornuftig å ha noen få (2-4) globale konstanter for farger. Font-family er en annen konstant jeg ser for meg blir brukt 'over alt'. 
Vil det være fornuftig å gjøre disse globale?